### ISSUE TITLE
Message for auto-play block is spammy

### ISSUE THREAD

**Issue by arisAlexis** [brave/ad-block#198]
The message "autoplay was blocked on this page" pops up several times in thee same page as for example on Facebook since it has a lot of video blocking meaning that you see this coming and going all the time. It's disturbing visually.Either just show it once or just show some X somewhere on the toolbar with more info.

**Comment by arisAlexis** [comment]
duplicate of browser/#3751
